src/clientSideScene/ClientSideSceneComp.tsx

```diff
@@ -76,9 +77,24 @@ export const ClientSideScene = ({
     }
   }, [])
 
+  let cursor = 'default'
+  if (state.matches('Sketch')) {
+    if (
+      context.mouseState.type === 'isHovering' &&
+      getParentGroup(context.mouseState.on, [ARROWHEAD, EXTRA_SEGMENT_HANDLE])
+    ) {
+      cursor = 'move'
+    } else if (context.mouseState.type === 'isDragging') {
+      cursor = 'grabbing'
+    } else {
+      cursor = 'crosshair'
+    }
+  }
+
```

Comment:
very js centric way of setting the cursor obviously, if we end up doing an overlay instead with HTML, we can dump this code for CSS on those elements.

Reply:
Totally fine with it maybe forever. With such a JS-centric app dropping this into CSS would force developers to hunt down where the cursor is set, I see why that could be a pain